### DISCUSSION TITLE
How to set a "fake progress bar" on the VIdeo Playing

### DISCUSSION THREAD

**Discussion by rodrigobg** [sampotts/plyr#2681] [Ideas]
Hello.
How can I add a fake progress bar while the video runs in muted autoplay?
As it happens on this site: https://pandavideo.com.br

Any ideas, or anyone with a good heart who knows how to implement?
I thank the help of all you.

**Comment by ashvaniskumar** [comment]
@rodrigobg I do not know if you have found the solution or not but i have done it tremendous times. I developed the solution for a client and it is doing magic. You can check it in action on these sites 
https://getnitricboost.com/discovery
https://buytc24.com/discovery

The second site also has custom Resume and Restart functionality with the custom Play and Pause Image button.

You can contact me on skype think4success1 for more info.
